rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -5960,13 +6052,21 @@ declare export class TransactionBuilderConfigBuilder {
   fee_algo(fee_algo: LinearFee): TransactionBuilderConfigBuilder;
 
   /**
+   * !!! DEPRECATED !!!
+   * Since babbage era cardano nodes use coins per byte. Use '.data_cost' instead.
    * @param {BigNum} coins_per_utxo_word
    * @returns {TransactionBuilderConfigBuilder}
    */
   coins_per_utxo_word(
     coins_per_utxo_word: BigNum
   ): TransactionBuilderConfigBuilder;
 
+  /**
+   * @param {DataCost} data_cost
+   * @returns {TransactionBuilderConfigBuilder}
+   */
+  data_cost(data_cost: DataCost): TransactionBuilderConfigBuilder;
```

Comment:
Since `DataCost` is just a wrapper for alternatives `coins_per_utxo_word` and `coins_per_utxo_byte` and we are deprecating already existing `coins_per_utxo_word` here - I'd rather just create new setter `.coins_per_utxo_byte` and point to that one from the deprecated setter, and then the `DataCost` structure can be used internally.